### PR TITLE
feat(cloud notifications) : Implement frontend support for adding BV in cloud notifications

### PR DESCRIPTION
**Pull request by Med1233**
## Description

**Fixes** # [(issue)](https://centreon.atlassian.net/browse/MON-19627)

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [x] 23.10.x (master)

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: centreon/www/front_src/src/CloudNotificationsConfiguration/Listing/Listing.tsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    setIsPannelOpen(true);
+  };
+
+  const predefinedRowsSelection = [
+    {
+      label: 'activated',
+      rowCondition: (row): boolean => row?.isActivated
+    },
+    {
+      label: 'desactivated',
```
Comment (bdauria) [inline centreon/www/front_src/src/CloudNotificationsConfiguration/Listing/Listing.tsx:83]: english translation is deactivated and not desactivated

**Review by cg-tw — DISMISSED** [review]

**Review by bdauria — DISMISSED** [review]

File: centreon/www/front_src/src/CloudNotificationsConfiguration/Panel/FormInitialValues/initialValues.ts [hunk 2]
```diff
@@ -1,14 +1,26 @@
...
-} from '../translatedLabels';
+} from '../../translatedLabels';
+import { SlackIcon, EmailIcon, SmsIcon } from '../FormInputs/Channel/Icons';
+import { NotificationType } from '../models';
+import { emptyEmail, formatMessages, formatResource } from '../utils';
 
-import { SlackIcon, EmailIcon, SmsIcon } from './Channel/Icons';
-import { NotificationType } from './models';
-import { emptyEmail, formatMessages, formatResource } from './utils';
+const formatBV = (isBamModuleInstalled, resources): object => {
```
Comment (bdauria) [inline centreon/www/front_src/src/CloudNotificationsConfiguration/Panel/FormInitialValues/initialValues.ts:12]: use object parameter instead

**Comment by Thebarda** [comment]
Is this PR still relevant?

**Comment by Med1233** [comment]
> Is this PR still relevant?

Unfortunately, I had to rebase every time before reaching the sufficient number of approvals

**Review by Thebarda — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**